### DISCUSSION TITLE
How to fine-tune couple

### DISCUSSION THREAD

**Discussion by imJunaidAfzal** [TheLastBen/fast-stable-diffusion#1108] [Q&A]
As we fine-tune face of men and women, how can we fine-tune a couple?
do I need both in same picture or separate images both?

**Comment by TheLastBen** [comment]
use separate images, rename the images of the man to a unique identifier and of the woman to another unique identifier

**Reply by imJunaidAfzal** [reply]
> use separate images, rename the images of the man to a unique identifier and of the woman to another unique identifier

I have tried this using 20 images for both male and female, using following settings:
### settings
train1
```
Images for both: 20
unet = 1200
textenc = 830
```
train2
```
Images for both: 20
unet = 2500
textenc = 1600
```
train3
```
Images for both: 20
unet = 4000
textenc = 2800
```
but still its unable to create accurate faces. Can you please suggest some settings for better results?

**Reply by TheLastBen** [reply]
what version of the model ? 1.5 or v2 ?

**Reply by imJunaidAfzal** [reply]
```
model = runwayml 1.5
vae = stabilityai/sd-vae-ft-mse

```

**Reply by TheLastBen** [reply]
the vae is already included in the 1.5, no need to specify it in the code.

set the Unet learning rate to 5e6 and reduce the total steps, keep the text encoder total steps to 350, once it's trained at this number of steps, keep it to 0 when resuming.

**Reply by imJunaidAfzal** [reply]
Thank you! I will try this setting.
